src/stream.c

```diff
@@ -383,7 +383,7 @@ int stream_alloc(struct stream **sp, const struct stream_param *prm,
 	s->rtcp  = s->cfg.rtcp_enable;
 
 	if (prm->use_rtp) {
-		err = stream_sock_alloc(s, call_af(call));
+		err = stream_sock_alloc(s, call ? call_af(call) : AF_INET);
```

Comment:
I know this is limited to use `AF_INET` family when `call` is not used, yet I find it a good trade-off.